### DISCUSSION TITLE
Worktree contains unstaged changes

### DISCUSSION THREAD

**Discussion by PaulRBerg** [lindell/multi-gitter#443] [Q&A]
## Error

I've just used Multi Gitter for the first time, and I got this error:

```text
Worktree contains unstaged changes:
  PaulRBerg/foundry-template
```

I don't understand how this is possible. Aren't all repositories cloned afresh? Does Multi Gitter somehow check if there is already a local clone of the repository, and, if yes, does it not create a new clone?

I happen to have a clone of that repository on my local machine, but there are no unstaged changes there.

## More Context

This is the command I've tried to run:

```sh
multi-gitter run ./update-funding.sh \
--branch docs/update-funding-url \
--commit-message "docs: update crypto funding URL" \
--dry-run \
--log-level=debug \
--repo "PaulRBerg/foundry-template"
```

And the script:

```sh
#!/usr/bin/env bash

rm ./.github/FUNDING.yml
echo -e "custom: \"https://omo.so/prberg\"\ngithub: \"PaulRBerg\"" > ./.github/FUNDING.yml
```

## Environment

- macOS 14.0
- git 2.41.0
- multi-gitter version: 0.49.0

**Comment by lindell** [comment]
Can you try to run the command with `--git-type=cmd`.

**Comment by adamhl8** [comment]
For anyone else having this problem, I found a solution that worked for me here: https://github.com/go-git/go-git/issues/511#issuecomment-1412524069

Basically, I had a symlink in my repo that was causing the issue. Ran `find -type l` and deleted it.
